include/fieldmap.h

```diff
@@ -50,14 +50,14 @@ u16 HiddenItemScriptNoToFlagId(u16 script);
 u16 HiddenItemScriptNoToHiddenItemIdx(u16 script);
 u8 sub_02040578(u16 script);
 BOOL GetHiddenItemParams(ScriptEnvironment *env, u16 script);
-HiddenItemResponse* AllocAndFetchNearbyHiddenItems(FieldSystem *fsys, HeapID heapId);
-void RunInitScript(FieldSystem *fsys);
-void StartMapLoadScript(FieldSystem *fsys, u16 script);
-BOOL TryStartMapScriptByType(FieldSystem *fsys, u8 type);
-BOOL MapSceneScriptCheck(FieldSystem *fsys);
+HiddenItemResponse* AllocAndFetchNearbyHiddenItems(FieldSystem *fieldSystem, HeapID heapId);
+void RunInitScript(FieldSystem *fieldSystem);
```

Comment:
We shold prolly go and clean up these function names as well

Reply:
out of scope (will get to this in a later PR)